cmd/kubeadm/app/phases/etcd/local_test.go

```diff
@@ -54,34 +54,67 @@ func TestGetEtcdPodSpec(t *testing.T) {
 }
 
 func TestCreateLocalEtcdStaticPodManifestFile(t *testing.T) {
-
 	// Create temp folder for the test case
 	tmpdir := testutil.SetupTempDir(t)
 	defer os.RemoveAll(tmpdir)
 
-	// Creates a Master Configuration
-	cfg := &kubeadmapi.InitConfiguration{
-		ClusterConfiguration: kubeadmapi.ClusterConfiguration{
-			KubernetesVersion: "v1.7.0",
-			Etcd: kubeadmapi.Etcd{
-				Local: &kubeadmapi.LocalEtcd{
-					DataDir: "/var/lib/etcd",
-					Image:   "k8s.gcr.io/etcd",
+	var tests = []struct {
```

Comment:
Another nit for consistency with the rest of kubeadm codebase

Typically we are storing only input/output variables of the test cases in this variable, not the test execution logic.

In this case  in the tests input is cfg, while the test output is expectedError.

The run function should go into the for loop,

Reply:
Updated! :)